Ginger/GingerCoreNETUnitTest/GingerRunnerTests/GingerRunnerTest.cs

```diff
@@ -265,7 +285,83 @@ public void TestRunsetConfigBFVariables()
 
         }
 
-        
+
+        [TestMethod]
+        public void TestDyanamicRunsetOprations()
+        {
+            RunSetConfig runSetConfigurations = CreteRunsetWithOperations();
+
+            GMR = new RunsetExecutor();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make the enclosing instance method 'static' or remove this set on the 'static' field.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4629760)